idaes/commands/extensions.py

```diff
@@ -233,11 +223,21 @@ def _write_hash(fp, pack, plat):
         fp.write("\n")
 
     with open(hfile, "w") as f:
-        for plat in idaes.config.basic_platforms():
+        for plat in idaes.config.base_platforms:
             for pack in ["solvers", "lib"]:
                 _write_hash(f, pack, plat)
-        for plat in idaes.config.basic_platforms():
+        for plat in idaes.config.base_platforms:
             for pack, sp in idaes.config.extra_binaries.items():
                 if plat not in sp:
                     continue
                 _write_hash(f, pack, plat)
+
+@cb.command(name="bin-platform", help="Show the compatible binary build.")
+def bin_platform():
+    fd, arch = idaes.util.download_bin._get_file_downloader(False, None)
+    try:
+        platform = idaes.util.download_bin._get_platform(fd, "auto", arch)
+    except idaes.util.download_bin.UnsupportedPlatformError:
+        click.echo("No compatible binaries found.")
+    else:
```

Comment:
Probably not that important, but does this really need to be in an `else` clause? Wouldn't it also work if this was in the `try` part?